src/components/LandingPage/LandingPage.css

```diff
@@ -0,0 +1,101 @@
+#landing-page-container {
+    display: flex;
+    justify-content: center;
+    max-width: 100%;
+}
+
+#info-section {
+    display: flex;
+    flex-direction: column;
+    max-width: 40%;
+    flex: 1;
+    margin-top: 5%;
+    margin-right: 3%;
+}
+
+#title {
+    font-weight: 700px;
+    color: #000000;
+    font-size: 54px;
+    font-style: normal;
+    letter-spacing: 0.035em;
+    line-height: 56px;
+    display: flex;
+    text-align: left;
+}
+
+#lab-description {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 24px;
+    letter-spacing: 0.04em;
+    display: flex;
+    text-align: left;
+    margin-top: 15px;
+}
+
+.btn_ubc_home {
+    display: flex;
+    width: 219px;
+    height: 42px;
+    background-color: #77B0F3;
+    font-weight: 700;
+    letter-spacing: 0.04em;
+    font-size: 15px;
+    line-height: 19px;
+    color: #ffffff;
+    text-align: center;
+    padding-bottom: 6px;
+    justify-content: center;
+    margin-top: 20px;
+}
+
+#landing-page-img-container {
+    margin-top: 75px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    overflow: hidden;
+    max-width: 50%;
+}
+
+#landing-page-img-placeholder {
+    background: #C4C4C4;
+    padding: 10px;
+    width: 491px;
+    height: 296px;
+    flex: 1;
+}
+
+@media (max-width:1025px){
+    #landing-page-container {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+
+    #info-section {
+        margin: auto;
+        justify-content: center;
+        max-width: 70%;
+    }
+
+    #title {
+        text-align: center;
+    }
+
+    .btn_ubc_home {
+        margin-left: 30px;
+    }
+    
+    #lab-description {
+        text-align: center;
+    }
+
+    #landing-page-img-container {
+        margin: auto;
+        margin-top: 50px;
+        justify-content: center;
+        overflow: hidden;
+    }
```

Comment:
I suppose adding a `height` attribute here would fix the following image size issue I got on my local:

<img width="251" alt="Screen Shot 2022-11-04 at 2 19 40 PM" src="https://user-images.githubusercontent.com/71746168/200076908-93c57a1d-23e3-4c49-b601-daaa027d29d9.png">

For example, adding the following line of code worked for me:

```suggestion
        overflow: hidden;
        height: 150px;
    }
```